src/tests/ftest/mpiio/llnl_mpi4py.py

```diff
@@ -26,7 +26,8 @@ def get_test_repo(self, name):
             str: python site-packages path to the test repository
         """
         test_repo = self.params.get(name, '/run/test_repo/')
-        for packages in site.getsitepackages():
+        # DAOS-14369: Always check the python3 install for test sources.
+        for packages in site.getsitepackages() + ["/usr/lib64/python3.6/site-packages"]:
```

Comment:
Why do we need to hard-code this? Shouldn't this path come from `site.getsitepackages()`?

Reply:
No. This is a deficiency in how the mpi tests are packaged. There's a lot of discussion on the subtleties in earlier iterations of this PR.

Reply:
@ashleypittman Can you (P&T) open a ticket describing the MPI tests packaging problem that is requiring this?  Might be helpful to include in the ticket the removal of this hard-coded path (a) as a test of the solution for that ticket and (b) just a reminder to remove this hard-coding as/when it's no longer necessary.

Reply:
https://daosio.atlassian.net/browse/DAOS-15602. Comment updated for the new ticket number as well.